spinners.json

```diff
@@ -1181,5 +1181,165 @@
 			"βββββρβ",
 			"ββββββρ"
 		]
+	},
+	"fingerdance": {
+		"interval": 160,
+		"frames": [
+			"☝️",
+			"✌️",
+			"🤘",
+			"🤟",
+			"🖖",
+			"🖐",
+			"✋",
+			"🤚",
+			"👆"
```

Comment:
Seems like these are not all the same rendered width. I tried it in `example.js` and the `Unicorn` text moves.

Reply:
I couldn't try it out in my shell yet, but at least [in the browser it looks fine](https://jsfiddle.net/sg2whe7t/), is this a shell-only thing, that different emoji have different width?

Reply:
Okay, so for some reason ☝️ and ✌️ are not shown in their "emoji form", in my shell at least. Removing them and adding a space after the 🖐 one fixed it for me.

![Kapture 2020-06-28 at 8 02 44](https://user-images.githubusercontent.com/20396367/85939434-e70c0d80-b915-11ea-96ef-8b8966d13a79.gif)

Reply:
I've ultimately removed these emoji, so this seems to be resolved.